CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.5.1 (2020-03-26)
+
+- Fixed the `package.json` file to list the correct `@skyux/list-builder` version as a peer dependency. [#59](https://github.com/blackbaud/skyux-select-field/pull/59)
```

Comment:
Why is this a fix instead of an update? And is it just me, or does this make it sound like it was previously listed as something other than a peer dependency? Would "Updated the `@skyux/list-builder` version." work?

Reply:
We forgot to update the version of an existing peer dependency when we needed to. This is a "fix" because it led to us not warning people when they hard an older version of the library that was not compatible with the latest version of this library.

Reply:
Had we done the update in the initial release of functionality I would agree that it is an "updated" but because we released a version with the incorrect version listed I think it should be a "fixed"

Reply:
That makes sense. I didn't realize. In that case, I think it's good as is. Thanks!